cli/compose/schema/schema_test.go

```diff
@@ -84,3 +84,16 @@ func TestValidatePlacement(t *testing.T) {
 
 	assert.NoError(t, Validate(config, "3.3"))
 }
+
+func TestValidateIsolation(t *testing.T) {
+	config := dict{
+		"version": "3.5",
+		"services": dict{
+			"foo": dict{
+				"image":     "busybox",
+				"isolation": "process",
```

Comment:
The actual validation of the _value_ for isolation is now handled by the daemon, correct? Given that this test is around validation (but only validating that _an_ "isolation" key is allows), I'd prefer the value to be different to show it's not being validated here. Something like `"some-isolation-value"`